atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -494,6 +506,7 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
+                    'config': config,
```

Comment:
Fixed and tested on docker 1.9 - this key is removed if config cannot be fetched